maro/simulator/scenarios/data_center/virtual_machine.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import List
+
+
+class VirtualMachine:
+    def __init__(self, id: str, req_cpu: int, req_mem: int, lifetime: int):
+        # Requirement
+        self.id: str = id
+        self.req_cpu: int = req_cpu
+        self.req_mem: int = req_mem
+        self.lifetime: int = lifetime
```

Comment:
what is the exact meaning of the ``lifetime`` ?

Reply:
deletion time - creation time